tests/storage/backendstorage.go

```diff
@@ -77,9 +77,6 @@ var _ = SIGDescribe("[Serial]Backend Storage", Serial, func() {
 				break
 			}
 		}
-		if storageClass == "" {
-			Skip("Failed to find a suitable storage class") // See TODO above
```

Comment:
Should we fail here?

Reply:
I don't think so, this was intended as a skip like the TODO mentioned

Reply:
Right, the intention of leaving explicit Fail would be to inform user that they messed up when they were selecting tests to run... (they forget to use the decorator)

Reply:
So the idea is that the person invocating the tests is completely responsible for selecting which tests run, but we could add a fail here as safeguard

Reply:
Correct